### PR TITLE
Add the release number in the manifest and change the staging ami as per the new ODFE structure.

### PR DESCRIPTION
**Pull request by sreekarjami**
*Issue #, if available:*

*Description of changes:*
Add the release number in the manifest and change the staging ami as per the new ODFE structure.

*Test Results:*
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/575122905

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: .github/workflows/staging-build-ami.yml [hunk 1]
```diff
@@ -17,15 +17,17 @@ jobs:
...
       env:
         AWS_DEFAULT_REGION: "us-east-1"
         AWS_ACCESS_KEY_ID: ${{ secrets.ODFE_AMI_AWS_ACCESS_KEY_ID }}
         AWS_SECRET_ACCESS_KEY: ${{ secrets.ODFE_AMI_AWS_SECRET_ACCESS_KEY }}
       run: |
         #!/bin/bash
-        export APT_OSS_version=`./bin/version-info --es`
-        OD_version=`./bin/version-info --od`
+        export APT_OSS_version=`./release-tools/scripts/version-info --es`
+        OD_version=`./release-tools/scripts/version-info --od`
```
Comment (gaiksaya) [inline .github/workflows/staging-build-ami.yml:30]: version-info.sh

File: release-tools/scripts/manifest.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ urls:
     prod: s3://artifacts.opendistroforelasticsearch.amazon.com/
     #snapshots: s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/
     releases: s3://staging.artifacts.opendistroforelasticsearch.amazon.com/releases/
-    releases_final_build: 
+    releases_final_build: 573103324
```
Comment (gaiksaya) [inline release-tools/scripts/manifest.yml:31]: There is a space at the end. Remove it just incase it causes path issues

Comment (peterzhuamazon) [reply release-tools/scripts/manifest.yml:31]: Is it rc-number or just number?

Comment (sreekarjami) [reply release-tools/scripts/manifest.yml:31]: Just number.

File: .github/workflows/staging-build-ami.yml (hunk 1 above)
Comment (sreekarjami) [reply .github/workflows/staging-build-ami.yml]: Done.

File: release-tools/scripts/manifest.yml (hunk 2 above)
Comment (peterzhuamazon) [reply release-tools/scripts/manifest.yml]: ok

**Review by peterzhuamazon — APPROVED** [review]

**Review by gaiksaya — APPROVED** [review]

**Pull request merged**